packages/emulators/standalone/suyu/config/SD865/qt-config.ini

```
@@ -0,0 +1,33 @@
+[Controls]
+keyboard_enabled=false
+player_0_button_a\default=false
+player_0_button_a="pad:0,button:1,port:0,guid:03000000202000000130000001000000,engine:sdl"
+player_0_button_b="pad:0,button:0,port:0,guid:03000000202000000130000001000000,engine:sdl"
+player_0_button_x="pad:0,button:2,port:0,guid:03000000202000000130000001000000,engine:sdl"
+player_0_button_y="pad:0,button:3,port:0,guid:03000000202000000130000001000000,engine:sdl"
+player_0_button_lstick="engine:sdl,port:0,guid:03000000202000000130000001000000,button:9"
+player_0_button_rstick="engine:sdl,port:0,guid:03000000202000000130000001000000,button:10"
+player_0_button_l="engine:sdl,port:0,guid:03000000202000000130000001000000,button:4"
+player_0_button_r="engine:sdl,port:0,guid:03000000202000000130000001000000,button:5"
+player_0_button_zl="engine:sdl,invert:+,port:0,guid:03000000202000000130000001000000,axis:6,threshold:0.500000"
+player_0_button_zr="engine:sdl,invert:+,port:0,guid:03000000202000000130000001000000,axis:7,threshold:0.500000"
+player_0_button_plus="engine:sdl,port:0,guid:03000000202000000130000001000000,button:7"
+player_0_button_minus="engine:sdl,port:0,guid:03000000202000000130000001000000,button:6"
+player_0_button_dleft="engine:sdl,port:0,guid:03000000202000000130000001000000,button:13"
+player_0_button_dup="engine:sdl,port:0,guid:03000000202000000130000001000000,button:11"
+player_0_button_dright="engine:sdl,port:0,guid:03000000202000000130000001000000,button:14"
+player_0_button_ddown="engine:sdl,port:0,guid:03000000202000000130000001000000,button:12"
+player_0_button_slleft="engine:sdl,port:0,guid:03000000202000000130000001000000,button:4"
+player_0_button_srleft="engine:sdl,port:0,guid:03000000202000000130000001000000,button:5"
+player_0_button_home="engine:sdl,port:0,guid:03000000202000000130000001000000,button:8"
+player_0_button_screenshot="engine:sdl,port:0,guid:03000000202000000130000001000000,button:15"
+player_0_button_slright="engine:sdl,port:0,guid:03000000202000000130000001000000,button:4"
+player_0_button_srright="engine:sdl,port:0,guid:03000000202000000130000001000000,button:5"
+player_0_lstick="engine:sdl,port:0,guid:03000000202000000130000001000000,axis_x:0,offset_x:-0.000000,axis_y:1,offset_y:0.000000,invert_x:+,invert_y:+,deadzone:0.150000"
+player_0_rstick="engine:sdl,port:0,guid:03000000202000000130000001000000,axis_x:3,offset_x:-0.000000,axis_y:4,offset_y:0.000000,invert_x:+,invert_y:+,deadzone:0.150000"
```

Once correctly calibrated with "Calibrate Gamepad" in tools the deadzone can be set to 0 :)

```suggestion
player_0_lstick="engine:sdl,port:0,guid:03000000202000000130000001000000,axis_x:0,offset_x:-0.000000,axis_y:1,offset_y:0.000000,invert_x:+,invert_y:+,deadzone:0.000000"
player_0_rstick="engine:sdl,port:0,guid:03000000202000000130000001000000,axis_x:3,offset_x:-0.000000,axis_y:4,offset_y:0.000000,invert_x:+,invert_y:+,deadzone:0.000000"
```